api/go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/hashicorp/go-rootcerts v1.0.2
 	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.1
 	github.com/hashicorp/hcl v1.0.0
-	github.com/hashicorp/vault/sdk v0.3.0
+	github.com/hashicorp/vault/sdk v0.4.1
```

Comment:
What changed between 0.4.0 and 0.4.1?

Reply:
I just mis-tagged 0.4.0 to not include the "v" prefix, so we wanted to release 0.4.1 to avoid any mod cache issues with 0.4.0.